CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 All notable changes to `dash-player` will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+Changes to `dash-player` were sponsored by Volkswagen's Center of Excellence.
```

Comment:
What is it actually called? Usually a Center of Excellence has more to its name... is it this: https://www.volkswagenag.com/en/news/stories/2019/09/battery-cell-assembly--pilot-line-started.html - the Volkswagen Center of Excellence for Battery Cells?

Reply:
@alexcjohnson yeah I just emailed Julian from VW to ask this - waiting on his reply before I change anything